### DISCUSSION TITLE
Bottom custom html editing

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#439] [General]
I edited the bottom section from the custom html page but it does not seem to apply it. Any ideas?

**Comment by rejetto** [comment]
i just made a test, added a word, and i seet it at bottom of the page.
try providing more info, screenshots

**Comment by Stoufs** [comment]
I added the word test and refreshed it and didn't do anything. If I add html, the page is blank.

**Comment by Stoufs** [comment]
All the other sections work fine except bottom.

**Comment by rejetto** [comment]
screenshots

**Comment by Stoufs** [comment]
<img width="1275" alt="image" src="https://github.com/rejetto/hfs/assets/154001951/dac4159b-3765-494e-8071-787e1ff8fe22">

and my admin custom page

<img width="797" alt="image" src="https://github.com/rejetto/hfs/assets/154001951/1675b0f4-bece-4114-8713-6e2f72f203a5">

**Reply by Stoufs** [reply]
As you can see, I added top custom wording and that worked and bottom, nothing.

**Reply by Stoufs** [reply]
This is in my custom file.

[afterMenuBar]
content here* <-- works

[bottom]
test 3 <--- doesn't

**Comment by rejetto** [comment]
did you try to scroll?
bottom is not for "bottom of the screen", but bottom of the file, and because of the how the layout is done it visible only after scrolling.
maybe "after list" suites your needs best

**Reply by Stoufs** [reply]
Just tried after list and that doesn't seem to work either. The error pages, after menu seem to be ok but these two do not

**Reply by Stoufs** [reply]
ok the after list did show up wayyyyyyyyyyyy at the bottom. How come so much spacing between list and that lol

**Comment by rejetto** [comment]
maybe you had some cache problem. When you reload, hold CTRL to be sure to avoid cache.

to place stuff at the right place you probably need html/css knowledge

**Comment by rejetto** [comment]
anyway, i see that the names top/bottom are misleading.
I don't think it can be made totally clear for anybody, because you will still need html knowledge to understand what these sections are, but i will use these names that are a bit clearer
<img width="221" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/0c380b9f-afa8-40d3-8754-c3424240189c">

**Reply by PH5HP** [reply]
Hi,

I was wondering also what the positions of the various sections are. Maybe you could provide a screenshot in the WIKI?

Kind regards,
Henk - PH5HP - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
yes, but everyone is invited at contributing on the wiki. There's so much missing, especially the basics

**Comment by rejetto** [comment]
i want to give some detail about the spacing thing.
That spacing is introduced by design by a css rule (specifically, flex:1 on the UL element), that you can override to your liking.
Setting it to 0, or any other value, like unset, will stop it.
I know this stuff is not really for everyone. To get what you want you need to know html and css, and use the tools of the browser (F12).
